include/string.cpp

```diff
@@ -0,0 +1,223 @@
+// Copyright 2020 Merzlov Nikolay merzlovnik@mail.ru
+#include "string.hpp"
+
+String::~String() {
+    delete[] Data;
+}
+
+String::String() : size(0) {
+    Data = new char[size];
+}
+
+String::String(const String &rhs) : size(rhs.size) {
+    Data = new char[size];
+    for (unsigned int i = 0; i < size; ++i) {
+        Data[i] = rhs[i];
+    }
+}
+
+String::String(const char *data) : size(0) {
+    const char *copy_str = data;
+    for (unsigned int i = 0; copy_str[i] != '\000'; ++copy_str)
+        ++size;
+    Data = new char[size];
+    for (unsigned int i = 0; i < size; ++i)
+        Data[i] = data[i];
+}
+
+void String::Resize(size_t new_size) {
+    String copy_str(*this);
+    delete[]Data;
+    Data = new char[new_size];
+    for (unsigned int i = 0; i < copy_str.size; ++i)
+        Data[i] = copy_str.Data[i];
+}
+
+String &String::operator=(const String &rhs) {
+    if (&rhs != this) {
+        Data = new char[rhs.size];
+        size = rhs.size;
+        for (unsigned int i = 0; i < size; ++i)
+            Data[i] = rhs[i];
+    }
+    return *this;
+}
+
+class String &String::operator+=(const class String &rhs) {
+    size = size + rhs.size;
```

Comment:
Unobvious code, use:
```suggestion
    size = this->size + rhs.size;
```
or
```suggestion
    newSize = size + rhs.size;
```